Source/ModuleKISInventory.cs

```diff
@@ -546,7 +546,7 @@ public enum InventoryType {
   public Rect guiMainWindowPos;
   Rect guiDebugWindowPos = new Rect(0, 50, 500, 300);
   KIS_IconViewer icon = null;
-  Rect defaultEditorPos = new Rect(Screen.width / 3, 40, 10, 10);
+  //Rect defaultEditorPos = new Rect(Screen.width / 3, 40, 10, 10);
```

Comment:
The code which is not needed should be simply removed. If there is a reason to keep the commented line, then the reason should be explained in the comment. Here and in the other changes of this PR.

Reply:
Ok, cool. We're of the same mind on that, actually.